### PR TITLE
Package rename

### PR DESCRIPTION
**Pull request by angelo-hub**
Renames the package and uses shiftsmart/@shiftsmartinc

### CODE REVIEW

File: android/src/main/java/com/shiftsmart/RNZenDeskSupport/HelpCenterActivityBuilder.java [hunk 1]
```diff
@@ -55,25 +55,25 @@ SupportActivityBuilder withOptions(ReadableMap options) {
       if (options.hasKey("withContactUsButtonVisibility")) {
         switch(options.getString("withContactUsButtonVisibility")) {
           case "OFF":
-            withContactUsButtonVisibility(ContactUsButtonVisibility.OFF);
```
Comment (visormatt) [inline android/src/main/java/com/shiftsmart/RNZenDeskSupport/HelpCenterActivityBuilder.java]: Are we sure we need to swap all these to manual booleans? Seems like they could be some `configuration` values set via the ZenDesk UI 🤔

File: android/src/main/java/com/shiftsmart/RNZenDeskSupport/RNZenDeskSupportModule.java [hunk 2]
```diff
@@ -20,18 +20,20 @@
...
-import com.zendesk.sdk.model.request.CustomField;
-import com.zendesk.sdk.model.request.CreateRequest;
-import com.zendesk.sdk.network.impl.ZendeskConfig;
-import com.zendesk.sdk.network.RequestProvider;
-import com.zendesk.service.ErrorResponse;
-import com.zendesk.service.ZendeskCallback;
+import zendesk.support.request.RequestActivity;
+import zendesk.support.requestlist.RequestListActivity;
+import zendesk.support.CustomField;
+import com.zendesk.*;
```
Comment (visormatt) [inline android/src/main/java/com/shiftsmart/RNZenDeskSupport/RNZenDeskSupportModule.java]: We should also just sanity check our application size, hopefully the wildcard imports don't bloat things too much. Not a deal breaker for the hot fix, but we should validate it

File: android/src/main/java/com/shiftsmart/RNZenDeskSupport/HelpCenterActivityBuilder.java [hunk 3]
```diff
@@ -1,17 +1,21 @@
-package com.robertsheao.RNZenDeskSupport;
```
Comment (visormatt) [inline android/src/main/java/com/shiftsmart/RNZenDeskSupport/HelpCenterActivityBuilder.java:1]: All references are left are docs 👍

**Review by visormatt — APPROVED** [review]
A lot of this is outside of my realm of experience but its looking good. Just a couple of comments, lmk. Otherwise we can get this merged into main, and LMK if you need a hand with the publishing step

File: android/src/main/java/com/shiftsmart/RNZenDeskSupport/RNZenDeskSupportModule.java (hunk 2 above)
Comment (angelo-hub) [reply android/src/main/java/com/shiftsmart/RNZenDeskSupport/RNZenDeskSupportModule.java]: I can keep narrowing it down and check if builds still work

File: android/src/main/java/com/shiftsmart/RNZenDeskSupport/HelpCenterActivityBuilder.java (hunk 1 above)
Comment (angelo-hub) [reply android/src/main/java/com/shiftsmart/RNZenDeskSupport/HelpCenterActivityBuilder.java]: for this version of the SDK it's defined on a ActivityBuilder level

File: android/src/main/java/com/shiftsmart/RNZenDeskSupport/RNZenDeskSupportModule.java (hunk 2 above)
Comment (angelo-hub) [reply android/src/main/java/com/shiftsmart/RNZenDeskSupport/RNZenDeskSupportModule.java]: was able to resolve this and remove all wildcards

**Pull request merged**